var/spack/repos/builtin/packages/axel/package.py

```diff
@@ -10,18 +10,14 @@ class Axel(AutotoolsPackage):
     """Axel is a light command line download accelerator for Linux and Unix"""
 
     homepage = "https://github.com/axel-download-accelerator/axel"
-    url      = "https://github.com/axel-download-accelerator/axel/archive/v2.16.1.tar.gz"
+    url      = 'https://github.com/axel-download-accelerator/axel/releases/download/v2.17.10/axel-2.17.10.tar.bz2'
 
-    version('2.16.1', sha256='64529add74df3db828f704b42d4ec3fcdacb8142c84f051f9213637c337e706c')
+    version('2.17.10', sha256='c0d26eba6b94945cd98c5b69ca6df2744639d17bfd49047ef51a8a48f067de10')
```

Comment:
We normally don't remove old versions. I would ask you to keep the old version (with the appropriate dependencies) and simply add the new one.

Reply:
done